transforms3d/taitbryan.py

```diff
@@ -68,19 +118,24 @@ def euler2mat(z, y, x):
     >>> yrot = -0.1
     >>> xrot = 0.2
     >>> M = euler2mat(zrot, yrot, xrot)
-    >>> M.shape == (3, 3)
```

Comment:
I seem to remember this gets round some Windows issue with shape being `(3L, 3L)`.